ansible/roles/postgresql/templates/pgbouncer.users.j2

```diff
@@ -0,0 +1 @@
+"{{ localsettings.PG_DATABASE_USER }}" "{{ localsettings.PG_DATABASE_PASSWORD }}"
```

Comment:
does this mean it will only clean the hqweb2postgres connnection? may want to add the devreadonly and/or sentry though those i'm sure are way less important